setup.py

```diff
@@ -8,6 +8,7 @@
 install_requirements = [
     "pgspecial>=1.11.8",
     "click >= 4.1",
+    "packaging>=20.9",
```

Comment:
Why do we need this one?

Reply:
Without it, we see the following error when we try to run pgcli after installation.

_Traceback (most recent call last):
  File "/usr/local/bin/pgcli", line 5, in <module>
    from pgcli.main import cli
  File "/usr/share/pgcli/lib/python3.7/site-packages/pgcli/main.py", line 56, in <module>
    from .pgtoolbar import create_toolbar_tokens_func
  File "/usr/share/pgcli/lib/python3.7/site-packages/pgcli/pgtoolbar.py", line 1, in <module>
    from packaging.version import parse as parse_version
ModuleNotFoundError: No module named 'packaging'_

Reply:
It looks like PR #1246 added in the packaging module.

Reply:
It was a mistake that is addressed in #1252. Packaging is a build dependency, but should not be needed at run time.

Reply:
> It was a mistake that is addressed in #1252. Packaging is a build dependency, but should not be needed at run time.

I'm certainly not qualified to question you but maybe you can advise how I can do the build differently.  I am building from the latest master branch.  When it builds, I then install the .rpm or .deb on a system.

If I don't include packaging in setup.py, it complains that it doesn't have it when I try to run it.  

Should I do something different?

Reply:
oh duh, I see that your pull request is still open :)

Ok, once you merge that I'll update this pull request.